.vscode/tasks.json

```diff
@@ -10,38 +10,51 @@
       "problemMatcher": {
         "owner": "typescript",
         "pattern": "$tsc",
-        "fileLocation": ["relative", "${workspaceRoot}/editors/code"]
+        "fileLocation": [
+          "relative",
+          "${workspaceRoot}/editors/code"
+        ]
       },
       "path": "editors/code/"
     },
     {
       "label": "Build Lsp",
       "type": "shell",
-      "command": "cargo build",
```

Comment:
`cargo build` is intentional here: it makes `F5` start an extension debug session with a freshly build analyzer

Reply:
Sorry, I will undo that change

Reply:
I see it now in the tasks.json. I hadn't really looked at that before, print debugging had been enough so far, but I will try it next time.